google-cloud-bigtable/lib/google/cloud/bigtable/table.rb

```diff
@@ -145,6 +136,25 @@ def cluster_states
         #
         # @return [Array<Google::Bigtable::ColumnFamily>]
         #
+        # @example
+        #   require "google/cloud/bigtable"
+        #
+        #   bigtable = Google::Cloud::Bigtable.new
+        #
+        #   table = bigtable.table("my-instance", "my-table", perform_lookup: true)
+        #
+        #   table.column_families.each do |cf|
+        #     p cf.name
+        #     p cf.gc_rule
+        #   end
+        #
+        #   # Get column family by name
+        #   cf1 = table.column_families.find_by_name("cf1")
```

Comment:
Does the method `ColumnFamilyMap#find_by_name ` exist?

Reply:
No, this example is one of two that are skipped pending follow-up API changes.

Reply:
K, I didn't see that these were skipped.

Reply:
See #4133 for follow-up that will should changes to this example.